### PR TITLE
Touch up docs after daml2js renaming

### PR DESCRIPTION
**Pull request by shayne-fletcher**
Fixing up user docs in light of `daml2js` renaming.

### CODE REVIEW

File: docs/source/daml2js/index.rst [hunk 1]
```diff
@@ -1,44 +1,42 @@
...
+Here's a complete example on a project built from the standard "skeleton" template.
 
 .. code-block:: bash
    :linenos:
 
    daml new my-proj skeleton # Create a new project based off the skeleton template
    cd my-proj # Enter the newly created project directory
    daml build  # Compile the project's DAML files into a DAR
-   daml codegen ts .daml/dist/my-proj-0.0.1.dar -o daml2js # Generate script bindings from the DAR
+   daml codegen js -o daml2js .daml/dist/my-proj-0.0.1.dar # Generate JavaScript packages in the daml2js directory
```
Comment (hurryabit) [inline docs/source/daml2js/index.rst:32]:
```suggestion
   daml codegen js -o dam.js .daml/dist/my-proj-0.0.1.dar # Generate JavaScript packages in the daml.js directory
```
and adapt wherever we refence this directory.

Comment (shayne-fletcher) [reply docs/source/daml2js/index.rst:32]: ok

**Comment by shayne-fletcher** [comment]
Will do

**Comment by shayne-fletcher** [comment]
Rebased.

**Comment by shayne-fletcher** [comment]
And rebased again. What's going on?!?

File: docs/source/daml2js/index.rst [hunk 2]
```diff
@@ -233,7 +231,7 @@ Here is a DAML template of a basic 'IOU' contract.
          do
            create this with owner = newOwner
 
-``daml2js`` generates types for each of the choices defined on the template as well as the template itself.
+The ``daml codgen js`` command generates types for each of the choices defined on the template as well as the template itself.
```
Comment (rohanjr) [inline docs/source/daml2js/index.rst:234]: codegen typo

Comment (shayne-fletcher) [reply docs/source/daml2js/index.rst:234]: Thanks!

**Pull request merged**